lib/elastic_apm/normalizers/rails/active_record.rb

```diff
@@ -11,6 +11,8 @@ class SqlNormalizer < Normalizer
 
         TYPE = 'db'
         ACTION = 'sql'
+        SKIP_NAMES = %w[SCHEMA CACHE].freeze
+        UNKNOWN = 'unknown'.freeze
```

Comment:
This is already frozen with `frozen_string_literal: true`

Reply:
doh! Sorry, old habits.